tests/Functional/run.php

```diff
@@ -16,7 +16,7 @@
 
 try {
     $app = new Application($config, $logger);
-    echo json_encode($app->run(), JSON_PRETTY_PRINT);
```

Comment:
nahoře někde zmizel return a tady zmizí i výstup. To je docela BC break a čekal bych ho skoro v samostatném PR, než přilepený tady.

Reply:
Je to soucast prechodu na BaseComponent -> pridal jsem `print()` uz v dilcich `*Action()` metodach.